insights/parsers/ceph_osd_log.py

```diff
@@ -27,4 +27,18 @@ class CephOsdLog(LogFileOutput):
     """
     Provide access to Ceph OSD logs using the LogFileOutput parser class.
     """
-    pass
+    def get_after(self, timestamp, lines=None):
+        """
+        Get a list of lines after the given time stamp.
+
+        Parameters:
+            timestamp(datetime.datetime): log lines after this time are
+            returned.
+            lines(list): the list of log lines to search (e.g. from a get).
+            If not supplied, all available lines are searched.
```

Comment:
@PaulWay subsequent lines should be indented here.  You can see this in the generated documentation.